### PR TITLE
Move marketplace_provisioning topic up one level

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Move topic under Build an agent with SDK and add a new topic under Manage your Marketplace (Provisioning Marketplace credentials) that point to this page so that it is easier to find.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 0fe64f7 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-347.amplify-central.opendocs-builder.pcloud.axway.int |

**Review by robertengaxway — APPROVED** [review]
Looks good to me

File: content/en/docs/Overview/provider_journey.md [hunk 1]
```diff
@@ -27,4 +27,4 @@ The following steps summarize the provider journey within the Amplify API Manage
     * End of life an asset - see [Deprecate an asset](/docs/manage_asset_catalog/asset_management/#deprecate-an-asset) and [Archive an asset](/docs/manage_asset_catalog/asset_management/#archive-an-asset)
 
 6. Publish / Monetize the product - see [Publish to marketplace](/docs/manage_marketplace/publish_to_marketplace)
-7. Manage your consumer subscriptions - see [Manage your Marketplace provisioning](/docs/connect_manage_environ/connected_agent_common_reference/marketplace_provisioning)
+7. Manage your consumer subscriptions - see [Manage your Marketplace provisioning](/docs/connect_manage_environ/marketplace_provisioning)
```
Comment (cbordier) [inline content/en/docs/Overview/provider_journey.md:30]: I would prefer:
7. Automate subscriber credentials provisioning/deprovisioning - see ...

**Review by cbordier — APPROVED** [review]
Approved when my last comment is integrated

**Pull request merged**